util/test/start_test.py

```diff
@@ -470,8 +472,11 @@ def run_sub_test(test=False):
 def generate_graphs(test=False):
     if test:
         basedir = os.path.dirname(test)
-        remove_dot = test.replace(".chpl", "").replace(".test.c", "")
+        remove_dot = test.replace(".chpl", "")
+        remove_dot = remove_dot.replace(".test.c", "")
+        remove_dot = remove_dot.replace(".test.cpp", "")
         remove_dot = remove_dot.replace(".ml-test.c", "")
+        remove_dot = remove_dot.replace(".ml-test.cpp", "")
```

Comment:
Don't you want to replace .cpp before ones that end with .c?

Reply:
d'oh, thanks for pointing it out